components/raftstore/src/store/msg.rs

```diff
@@ -73,9 +73,8 @@ where
     }
 }
 
-pub type ReadCallback<S> = Box<dyn FnOnce(ReadResponse<S>) + Send>;
```

Comment:
Because `ReadCallback` is a trait now. `WriteCallbackExt` is a bad name, it's not an extension.

Reply:
Then rename the old ReadCallback with some other name. It's inconsistent that ExtCallback exists, but read/write callback does not have the alias, just because you use them somewhere else.